sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/RecognizeContentWithSelectionMarksAsync.java

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
why split this from the RecognizeContent sample? Is the output too crowded?

Reply:
Agree, better to have just one that shows everything... it will reduce the amount of sample files

Reply:
Yes. I can reduce the samples